src/index.js

```diff
@@ -507,7 +507,10 @@ export function initialize(env, user, specifiedOptions, platform, extraDefaults)
     });
   });
 
-  if (typeof options.bootstrap === 'string' && options.bootstrap.toUpperCase() === 'LOCALSTORAGE') {
+  if (typeof options.bootstrap === 'string' &&
+    // Support local or session storage names being passed as a bootstrap option.
+    ['SESSIONSTORAGE', 'LOCALSTORAGE'].includes(options.bootstrap.toUpperCase())
```

Comment:
I believe Array.includes() requires ES7 and I'm not sure if the transpiler will replace it with a portable method or not.